api/pacman-api-admin/src/main/java/com/tmobile/pacman/api/admin/domain/plugin/ContrastPluginRequest.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ *  Copyright 2023 Paladin Cloud, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License.  You may obtain a copy
+ *  of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ ******************************************************************************/
+package com.tmobile.pacman.api.admin.domain.plugin;
```

Comment:
I think we need to move new plugins out of the Pac-Man package and into some new paladincloud packs